course/src/main/java/org/openedx/course/presentation/container/CourseContainerFragment.kt

```diff
@@ -433,10 +439,20 @@ fun DashboardPager(
                             )
                         }
                     ),
+                    iapViewModel = koinViewModel(
```

Comment:
Why iapViewModel is viewModel, maybe we can change it to the interactor or manager? Benefits only in viewModelScope, but we can change functions with viewModelScope to suspend, or use another Scope. 
If we will move out of viewModel, we can reuse this code in another instances